content/docs/fragments.md

```diff
@@ -0,0 +1,144 @@
+---
+id: fragments
+title: Fragments
+permalink: docs/fragments.html
+---
+
+A common pattern in React is for a component to return multiple elements. React provides first-class `Fragment` support that lets you group a list of children without adding extra nodes to the DOM.
```

Comment:
"React provides first-class `Fragment` support that lets you" -> "Fragments let you"

Reply:
Let's remove the first sentence since it's repeated in the "Motivation" section